4 Elements 2 - Error after loading

### Describe the bug

The console throws an error after the game is loaded and you can't play

core/src/avm2/events.rs:428 Error dispatching event EventObject(EventObject { type: "load_game_error", class: game.api::ApiEvent, ptr: 0x5b286b0 }) to handler FunctionObject(FunctionObject { ptr: 0x5afb118, name: Ok("game::Game/onLoadGame()") }) : ArgumentError: Error #2008: Parameter align must be one of the accepted values.

### Expected behavior

load & play

### Content Location

https://data.gameflare.com/games/2532/8QspNCHbEwQTm6.swf

### Affected platform

Online demo

### Operating system

All

### Browser

All

### Additional information

_No response_

--- a/core/src/avm2/globals/flash/text/TextField.as
+++ b/core/src/avm2/globals/flash/text/TextField.as
@@ -7,69 +7,69 @@ package flash.text {
     public class TextField extends InteractiveObject {
         internal var _styleSheet:StyleSheet;
         internal var _useRichTextClipboard:Boolean;
-        
+
         public native function get alwaysShowSelection():Boolean;
         public native function set alwaysShowSelection(value:Boolean):void;
 
         public native function get autoSize():String;
         public native function set autoSize(value:String):void;
-        
+
         public native function get background():Boolean;
         public native function set background(value:Boolean):void;
-        
+
         public native function get backgroundColor():uint;
         public native function set backgroundColor(value:uint):void;
-        
+
         public native function get border():Boolean;
         public native function set border(value:Boolean):void;
-        
+
         public native function get borderColor():uint;
         public native function set borderColor(value:uint):void;
-        
+
         public native function get bottomScrollV():int;
-        
+
         public native function get condenseWhite():Boolean
         public native function set condenseWhite(value:Boolean):void
 
         public native function get defaultTextFormat():TextFormat;
         public native function set defaultTextFormat(value:TextFormat):void;
-        
+
         public native function get displayAsPassword():Boolean;
         public native function set displayAsPassword(value:Boolean):void;
-        
+
         public native function get embedFonts():Boolean;
         public native function set embedFonts(value:Boolean):void;
-        
+
         public native function get htmlText():String;
         public native function set htmlText(value:String):void;
-        
+
         public native function get length():int;
-        
+
         public native function get maxScrollH():int;
-        
+
         public native function get maxScrollV():int;
-        
+
         public native function get maxChars():int;
         public native function set maxChars(value:int):void;
 
         public native function get mouseWheelEnabled():Boolean
         public native function set mouseWheelEnabled(value:Boolean):void
-        
+
         public native function get multiline():Boolean;
         public native function set multiline(value:Boolean):void;
-        
+
         public native function get restrict():String;
         public native function set restrict(value:String):void;
-        
+
         public native function get scrollH():int;
         public native function set scrollH(value:int):void;
-        
+
         public native function get scrollV():int;
         public native function set scrollV(value:int):void;
-        
+
         public native function get selectable():Boolean;
         public native function set selectable(value:Boolean):void;
-        
+
         public function get styleSheet():StyleSheet {
             return this._styleSheet;
         }
@@ -77,44 +77,44 @@ package flash.text {
             this._styleSheet = value;
             stub_setter("flash.text.TextField", "styleSheet");
         }
-        
+
         public native function get text():String;
         public native function set text(value:String):void;
-        
+
         public native function get textColor():uint;
         public native function set textColor(value:uint):void;
-        
+
         public native function get textHeight():Number;
-        
+
         public native function get textWidth():Number;
-        
+
         public native function get type():String;
         public native function set type(value:String):void;
-        
+
         public function get useRichTextClipboard():Boolean {
             return this._useRichTextClipboard;
         }
-        
+
         public function set useRichTextClipboard(value:Boolean):void {
             this._useRichTextClipboard = value;
             stub_setter("flash.text.TextField", "useRichTextClipboard");
         }
-        
+
         public native function get wordWrap():Boolean;
         public native function set wordWrap(value:Boolean):void;
-        
+
         public native function get antiAliasType():String;
         public native function set antiAliasType(value:String):void;
-        
+
         public native function get gridFitType():String;
         public native function set gridFitType(value:String):void;
-        
+
         public native function get thickness():Number;
         public native function set thickness(value:Number):void;
-        
+
         public native function get sharpness():Number;
         public native function set sharpness(value:Number):void;
-        
+
         public native function get numLines():int;
 
         public native function get caretIndex(): int;
@@ -129,7 +129,7 @@ package flash.text {
         public native function replaceSelectedText(value:String):void;
         public native function replaceText(beginIndex:int, endIndex:int, newText:String):void;
         public native function setSelection(beginIndex:int, endIndex:int):void;
-        
+
         public function insertXMLText(beginIndex:int, endIndex:int, text:String, paste:Boolean = false):void {
             stub_method("flash.text.TextField", "insertXMLText");
         }
@@ -137,6 +137,16 @@ package flash.text {
         public function getCharIndexAtPoint(x:Number, y:Number):int {
             stub_method("flash.text.TextField", "getCharIndexAtPoint");
             return 0;
+        }
+
+        public function getLineLength(lineIndex:int):int {
+            stub_method("flash.text.TextField", "getLineLength");
+            return 0;
+        }
+
+        public function getLineText(lineIndex:int):String {
+            stub_method("flash.text.TextField", "getLineText");
+            return "";
         }
     }
 }

